### PR TITLE
style: fix rfc typos

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Fixes some typos found in doc.

### CODE REVIEW

**Review by wanglei4687 — APPROVED** [review]

File: docs/rfc/2022-07-05-mo-cluster.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ An operator deployment consists of:
   - multiple operator replicas would perform leader-election via k8s apiserver (backed by ETCD) and only one replica is working at a time;
 - RBAC policies that allows the operator to manage other k8s resources and cloud resource;
```
Comment (sukki37) [inline docs/rfc/2022-07-05-mo-cluster.md:160]: typo:
- cloud resource -> cloud resources
-  allows  -> allow

File: docs/rfc/2022-07-05-mo-cluster.md [hunk 2]
```diff
@@ -82,22 +82,22 @@ graph TD
 
 Physically, all the CRs are served by the kubernetes apiserver and all the controllers are running in goroutines of the operator process concurrently.
 
-Is is worth noting that `MatrixOneCluster` object is not necessary to create a mo-cluster, one can create `LogSet`, `CNSet` and `DNSet` separately to form a fully functional cluster. Such decoupling enables the following capabilities without losing the simplicity for end user:
+It is worth noting that `MatrixOneCluster` object is not necessary to create a mo-cluster, one can create `LogSet`, `CNSet` and `DNSet` separately to form a fully functional cluster. Such decoupling enables the following capabilities without losing the simplicity for end user:
 
 - advanced user can ignore `MatrixOneCluster` object perform fine-grained control over `*Set` directly;
 - it is trivial to support heterogenous cluster: one simply deploy several `*Set` objects with different Pod specification;
```
Comment (sukki37) [inline docs/rfc/2022-07-05-mo-cluster.md:88]: heterogenous -> heterogeneous

File: docs/rfc/2022-07-05-mo-cluster.md [hunk 3]
```diff
@@ -82,22 +82,22 @@ graph TD
...
 
 ### Cluster Deployment
 
-Once a `MatrixOneCluster` object is created in k8s-apiserver, it would be watched by the operator and the operator start reconciling.
+Once a `MatrixOneCluster` object is created in k8s-apiserver, it is watched by the operator and the operator starts reconciling.
 
 The `MatrixOneCluster` controller simply:
 
 - create a `LogSet` object, a `DNSet` object and a `CNSet` object;
 - wait them become ready;
```
Comment (sukki37) [inline docs/rfc/2022-07-05-mo-cluster.md:97]: wait for them to become ready

**Pull request merged**